lib/Widgets/SettingsSidebar.vala

```diff
@@ -33,6 +33,22 @@ public class Granite.SettingsSidebar : Gtk.ScrolledWindow {
     public Gtk.Stack stack { get; construct; }
 
     /**
+     * The name of the currently visible Granite.SettingsPage
+     */
+    public string visible_child_name {
+        get {
+            return ((SettingsSidebarRow) listbox.get_selected_row ()).name;
```

Comment:
Before returning that, check if the `get_selected_row` doesn't return `null`. If it did, return null also.